#Request string of face keyframes
string[] face_dofs
float64[] times
cordial_face/Keyframe[] frames